src/contentmanagerdelegate.cpp

```diff
@@ -164,6 +164,17 @@ void ContentManagerDelegate::paintButton(QPainter *p, const QRect &r, QString t)
     baseButton->style()->drawControl( QStyle::CE_PushButton, &button, p, baseButton.data());
 }
 
+void ContentManagerDelegate::paintBookDescription(QPainter *p, const QRect &r, const QModelIndex &index) const
+{
+    QRect nRect = r;
+    auto view = KiwixApp::instance()->getContentManager()->getView()->getView();
+
+    /* Align text to the x coordinate of column index 1, the Name column. */
+    nRect.setX(view->visualRect(index.parent().siblingAtColumn(1)).x());
+    nRect.setWidth(view->width());
+    p->drawText(nRect, Qt::AlignLeft | Qt::AlignVCenter, index.data(Qt::UserRole+1).toString());
+}
+
```

Comment:
This function was extracted as a result of refactoring but also contains a small change (`nRect.setX()`). Better do that in separate commits. Definitely, introducing `ContentManagerDelegate::paintBookDescription()` as a refactoring step doesn't fix the bug. Does `nRect.setX()` help to do it? Please split this commit into two or three so that the bug fix contains only contains changes that actually fix the bug.

Reply:
Yeah, it doesn't do the fix on its own. My mistake.

Reply:
Resolved. In the new version, we don't need this refactor anymore as it is removed.